samples/revoke_dataset_access.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def revoke_dataset_access(dataset_id, entity_id):
+
+    # [START bigquery_revoke_dataset_access]
+    from google.cloud import bigquery
+
+    # Construct a BigQuery client object.
+    client = bigquery.Client()
+
+    # TODO(developer): Set dataset_id to the ID of the dataset to fetch.
+    # dataset_id = 'your-project.your_dataset'
+
+    dataset = client.get_dataset(dataset_id)  # Make an API request.
+    entries = list(dataset.access_entries)
+
+    for entry in entries:
+        if entry.entity_id == entity_id:
+            entry.role = None
+            break
+
+    dataset.access_entries = entries
+
+    dataset = client.update_dataset(dataset, ["access_entries"])  # Make an API request.
+
+    full_dataset_id = "{}.{}".format(dataset.project, dataset.dataset_id)
```

Comment:
(nit)

It would probably be a tad more readable and concise it we used f-strings for string formatting, since we only support Python 3.6+.

Reply:
+1

Reply:
changed to f-strings in both main file and test!

Reply:
done